.github/workflows/kind.yml

```diff
@@ -256,15 +270,15 @@ jobs:
         uses: actions/download-artifact@v3
         with:
           name: clickhouse-monitor
-      - name: Load Theia image
+      - name: Load Theia clickhouse-monitor image
```

Comment:
Maybe merge this with the step Load Theia clickhouse-server image just as we do in test-upgrade-from-N-1?